server/remote/msgraph/call.go

```diff
@@ -11,12 +11,29 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strings"
 
 	"github.com/pkg/errors"
 	msgraph "github.com/yaegashi/msgraph.go/v1.0"
 )
 
-func (c *client) Call(method, path string, in, out interface{}) (responseData []byte, err error) {
+func (c *client) CallJSON(method, path string, in, out interface{}) (responseData []byte, err error) {
+	contentType := "application/json"
+	buf := &bytes.Buffer{}
+	err = json.NewEncoder(buf).Encode(in)
+	if err != nil {
+		return nil, err
+	}
+	return c.Call(method, path, contentType, buf, out)
+}
+
+func (c *client) CallURLEncodedForm(method, path string, in url.Values, out interface{}) (responseData []byte, err error) {
```

Comment:
0/5
1. Maybe just `CallFormPost` or even `CallForm`?
2. With the addition of these 2 methods, maybe hide `Call` as private? Seems redundant now.